### PR TITLE
Initial version of the netci.groovy for code coverage

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

**Comment by mmitche** [comment]
@dotnet-bot test ci please

**Comment by mmitche** [comment]
test ci please

**Comment by mmitche** [comment]
test ci please

File: netci.groovy [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+def static addPeriodicTrigger(def job, def cronString) {
+    job.with {
+        triggers {
+            cron(cronString)
+        }
+    }
+}
+
+// Define build string
+def codeCoverageBuildString = '''call "C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\Common7\\Tools\\VsDevCmd.bat" && build.cmd /p:Coverage=true'''
```
Comment (stephentoub) [inline netci.groovy]: Do we need to add /t:rebuild to force a rebuild/retest, or is that implicit in how this is used?

Comment (stephentoub) [inline netci.groovy]: Also, with the 14.0, it looks like this is assuming VS2015?

Comment (mmitche) [inline netci.groovy]: The workspace is wiped before run, so it's a clean enlistment.

**Comment by mmitche** [comment]
test ci please

**Comment by stephentoub** [comment]
I'm a groovy novice, but it looks good to me.

**Pull request merged**